app/models/package.rb

```diff
@@ -539,6 +539,10 @@ def dispatched_orders_package
     orders_packages.get_dispatched_records_with_order_id(order_id).first
   end
 
+  def donor_condition_name
+    try(:donor_condition).try(:name_en) || try(:item).try(:donor_condition).try(:name_en)
+  end
```

Comment:
This feels very defensively programmed! Do we need all these 'trys'?

    donor_condition.try(:name_en) || item.try(...)